src/storage/compact_filter.cc

```diff
@@ -86,8 +87,13 @@ Status SubKeyFilter::GetMetadata(const InternalKey &ikey, Metadata *metadata) co
 }
 
 bool SubKeyFilter::IsMetadataExpired(const InternalKey &ikey, const Metadata &metadata) const {
+  // lazy delete to avoid race condition between command Expire and subkey Compaction
+  // Related issue:https://github.com/apache/incubator-kvrocks/issues/1298
+  //  `Util::GetTimeStamp() - 1000` means extending 1000s for expired items,
+  // or they will be recycled once they reach the expire time.
+  int64_t lazy_expired_ts = Util::GetTimeStamp() - 1000;
```

Comment:
```suggestion
  // lazy delete to avoid race condition between command Expire and subkey Compaction
  // Related issue:https://github.com/apache/incubator-kvrocks/issues/1298
  //  `Util::GetTimeStamp() - 60` means extending 1min for expired items,
  // or they will be recycled once they reach the expiration time.
  int64_t lazy_expired_ts = Util::GetTimeStamp() - 60;
```